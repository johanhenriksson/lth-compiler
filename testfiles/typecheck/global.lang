bool b;
int c;

int main() {
    b = true; 
    b = 1; // error
    c = 1;
    c = false; // error
}
