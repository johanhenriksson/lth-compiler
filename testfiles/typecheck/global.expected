Error at line 6,5: cannot assign 'b' of type bool, got int
Error at line 8,5: cannot assign 'c' of type int, got bool
